.gitignore

```diff
@@ -94,6 +94,7 @@ ossreadme*.txt
 *.mdf
 *.ldf
 *.cto
+*.vstman
```

Comment:
Generated vsix manifest during build that contains information about project/item templates in the vsix: https://msdn.microsoft.com/en-us/library/mt706510.aspx